wagtailio/areweheadlessyet/migrations/0001_initial.py

```diff
@@ -1,11 +1,11 @@
 # Generated by Django 3.2.8 on 2022-01-24 11:57
 
 import django.db.models.deletion
-from django.db import migrations, models
-
 import wagtail.blocks
 import wagtail.fields
 
+from django.db import migrations, models
+
 
```

Comment:
@zerolab this change looks a bit strange to me - I assume this happens because `import x` takes precedence over `from x import y`. I would prefer third-party imports stay in alphabetical order.

Is this intentional behavior by ruff or is this something we can have an opinion about?

Reply:
it is a ruff opinion. will have a look tommorrow whether we can influence that

Reply:
the best I could get it to is:

```diff
+ import django.db.models.deletion
from django.db import migrations, models
- import django.db.models.deletion

import wagtail.blocks
import wagtail.fields
```

using https://docs.astral.sh/ruff/settings/#lint_isort_force-sort-within-sections